src/coreclr/nativeaot/docs/compiling.md

```diff
@@ -49,7 +49,7 @@ Targeting win-arm64 on a Windows x64 host machine,
 
 To target win-arm64 on a Windows x64 host on an advanced scenario where the SDK support is not sufficient (note that these scenarios will generate warnings for using explicit package references), in addition to the `Microsoft.DotNet.ILCompiler` package reference, also add the `runtime.win-x64.Microsoft.DotNet.ILCompiler` package reference to get the x64-hosted compiler:
 ```xml
-<PackageReference Include="Microsoft.DotNet.ILCompiler; runtime.win-x64.Microsoft.DotNet.ILCompiler" Version="8.0.0-preview.2.22103.2" />
+<PackageReference Include="Microsoft.DotNet.ILCompiler; runtime.win-x64.Microsoft.DotNet.ILCompiler" Version="8.0.0-alpha.1.22609.9" />
```

Comment:
We can use [floating notation](https://learn.microsoft.com/en-us/nuget/concepts/package-versioning#version-ranges): `Version="8.0.0-*"`. It will pick up the highest version available (by semver metadata).

Reply:
Floating notation will lead to mismatched x64 vs. arm64 versions if the nuget restored happens to run right around the time a new build is getting published. It causes very hard to debug issues as the next paragraph explains.